napari/_qt/dialogs/qt_notification.py

```diff
@@ -351,8 +359,6 @@ def from_notification(
 
         from ...utils.notifications import ErrorNotification
 
-        actions = notification.actions
-
```

Comment:
I think that these lines are obsolete, so I remove them, but maybe there are some side effects?